plan/plan.go

```diff
@@ -0,0 +1,75 @@
+package plan
+
+// DNSRecord holds information about a DNS record.
+type DNSRecord struct {
+	// The hostname of the DNS record
+	DNSName string
```

Comment:
might make sense to change to just Name, since it is already **DNS**Record struct :P